frontend/ui/bidi.lua

```diff
@@ -126,6 +127,22 @@ function Bidi.mirroredUILayout()
     return Bidi._mirrored_ui_layout
 end
 
+-- This fuction can be used by document widgets to match a widget
+-- to the document layout instead of the UI layout.
+function Bidi.invert()
```

Comment:
I'd like to see "temporarily" in this comment :)
(Also please make the commit title a bit less dry than just `Bidi: invert` , ie. `Bidi: add Bidi.invert()`.)

Reply:
done